packages/front-end/components/EventWebHooks/EventWebHookList/EventWebHookList.tsx

```diff
@@ -40,16 +40,25 @@ export const EventWebHookList: FC<EventWebHookListProps> = ({
       ) : null}
 
       <div className="mb-4">
-        <div className="d-flex justify-space-between align-items-center">
+        <div className="d-flex">
```

Comment:
Small nit: I think we might want to add the `align-items-center` class back in - otherwise the `Beta` badge styling is a bit funky. 

<img width="492" alt="Screen Shot 2024-07-11 at 10 43 18 AM" src="https://github.com/growthbook/growthbook/assets/75274610/27c36ce4-b626-4824-a273-507219f29aaa">

Reply:
Thanks, I totally missed that one. Fixed!